.github/workflows/textlint.yml

```
@@ -0,0 +1,12 @@
+name: textlint for Japanese documentation
+on: [pull_request]
```

Perhaps we could limit the action to just changes to `~/content/ja`?
```suggestion
on:
  pull_request:
    paths:
      - 'content/ja/**'
```

---

yes think that would be better

---

Thank you so much!! :laughing: